src/data/content/design-system/componenti/popover.yaml

```diff
@@ -65,7 +65,9 @@ tabs:
             title: Quando usarlo
             headingLevel: 2
             text: |
-              _In stesura_
+              Il componente popover può essere utilizzato quando si desidera fornire informazioni aggiuntive o contestuali, senza interrompere la navigazione o un flusso di lavoro in corso.
```

Comment:
```suggestion
              Il componente Popover può essere utilizzato per fornire informazioni aggiuntive o contestuali, senza interrompere la navigazione o un flusso di lavoro in corso.
```

Reply:
scusa @danielenole, ma nella PR #1331 i nomi dei componenti sono stati portati in minuscolo, qui è corretto in maiuscolo 🤔